pkg/config/config.go

```diff
@@ -36,6 +36,12 @@ type Asset struct {
 	Path string
 }
 
+// Route define forwarding routes
+type Route struct {
+	Endpoint string `json:"endpoint"`
+	Location string `json:"location"`
```

Comment:
Maybe rename Location to Backend to use established proxy-server speak?

Reply:
👍